frontend/src/components/molecules/DetailView.tsx

```diff
@@ -29,32 +82,55 @@ const DetailView: React.FC = () => {
     setDevelopmentAreas(await getDevelopmentArea(selectedNode.id));
   };
 
+  const expandConnection = (connection: Node) => {
+    setSelectedConnection(connection);
+    setExpanded(true);
+  };
+
   useEffect(() => {
     loadData();
   }, [selectedNode]);
 
   return (
-    <Box bg="cyan.600" w="100%" p={6} color="white">
-      <Container w="60%" maxW="1000px">
-        <Heading as="h2" size="2xl" fontWeight="hairline">
-          {annotations.label.toUpperCase() || (selectedNode && selectedNode.name) || ''}
-        </Heading>
-        <Heading as="h3" size="md" my="4">
-          {annotations.description}
-        </Heading>
-        <Connections
-          connections={contributions}
-          titles={['Har positiv virkning til', 'har ingen etablerte positive påvirkninger enda']}
-        />
-        <Connections
-          connections={tradeOffs}
-          titles={['Har negativ virkning til', 'Har ingen etablerte negative påvirkninger enda']}
-        />
-        <Connections
-          connections={developmentAreas}
-          titles={['Har utviklingsområde til', 'Har ingen utviklingsområder']}
+    <Box spacing={10} bg="cyan.500" w="100%" px={10} py={6} color="white">
+      <Heading as="h2" size="2xl" fontWeight="hairline" textAlign="left" paddingBottom="5">
+        {annotations.label.toUpperCase() || (selectedNode && selectedNode.name) || ''}
+      </Heading>
+      <Flex justify="space-between">
+        <SlideInDrawer expanded={!expanded} width="40vw">
+          <Text fontSize="xl" mt="2">
+            {annotations.description}
+          </Text>
+        </SlideInDrawer>
+        <ContextDivider visible={!expanded} />
+        <AllConnections
+          contributions={contributions}
+          tradeOffs={tradeOffs}
+          developmentAreas={developmentAreas}
+          onClick={expandConnection}
         />
-      </Container>
+        <ContextDivider visible={expanded} />
+        <SlideInDrawer expanded={expanded} width="40vw">
+          <>
+            <Heading as="h3">
+              {`${annotations.label} har ${
+                selectedConnection && mapCorrelationToName(selectedConnection.correlation)
+              } korrelasjon til ${selectedConnection && selectedConnection.name}`}
+            </Heading>
+            <Text>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti porro saepe
+              laboriosam iure amet, atque, id ex asperiores tempora voluptatem totam necessitatibus.
+              A maiores laboriosam, pariatur earum perferendis distinctio dicta?
```

Comment:
yeet?

Reply:
I think it should stay until we have real data to put in there.